gallery/workbook/Azure Monitor.json

```diff
@@ -212,10 +212,10 @@
 			"name": "Insights",
 			"templates": [
 				{
-					"id": "Workbooks/Windows Virtual Desktop/WVD Insights",
+					"id": "Workbooks/Windows Virtual Desktop/AtScale/Overview",
 					"author": "Microsoft",
 					"description": "Diagnostics, performance, and usage for Windows Virtual Desktop",
```

Comment:
Azure instead of Windows on this line too

Reply:
Updated the description for the gallery.

Reply:
Looks like line 217 could still use an update though